libEDSsharp/CanOpenNodeExporter_V4.cs

```diff
@@ -23,6 +23,7 @@
 using System.Text;
```

Comment:
can you make the commit log putting the subject as a short line and the details in the body? https://chris.beams.io/posts/git-commit/

I'd also write it from the perspective of what was changed, not who changed it. Your attribution is attached to the git commit log.

Also, did we really revert back to the original method? I don't believe we did that entirely so we shouldn't say it.

Also you mention an exception but you don't mention the specific fault case.